AppleCider/hyrax_datasets/spectra_dataset.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+import numpy as np
+import torch
+from hyrax.data_sets.data_set_registry import HyraxDataset
+from torch.utils.data import Dataset
+
+
+class SpectraData(HyraxDataset, Dataset):
+    def __init__(self, config, data_location=None):
+        super().__init__(config)
+
+        data_table = torch.load(config["model_inputs"]["data"]["file_path"])
```

Comment:
If you update the `max_config.toml` file to use `data_location`, then this line can become
```suggestion
        data_table = torch.load(data_location)
```